Containerfile

```diff
@@ -44,7 +44,7 @@ RUN /tmp/build.sh && \
 COPY --from=cgr.dev/chainguard/kubectl:latest /usr/bin/kubectl /usr/bin/kubectl
 COPY --from=cgr.dev/chainguard/cosign:latest /usr/bin/cosign /usr/bin/cosign
 
-RUN curl -Lo ./kind "https://kind.sigs.k8s.io/dl/v0.17.0/kind-$(uname)-amd64"
+RUN curl -Lo ./kind "https://kind.sigs.k8s.io/dl/v0.20.0/kind-$(uname)-amd64"
```

Comment:
Just a thought, any reason to not download the latest release automatically?

https://github.com/kubernetes-sigs/kind/releases/latest/download/kind-linux-arm64
This always points to the `latest` release and would fit with the model of automated updates.  The day after a new release is published, it's included in the image